starboard/build/toolchain/cobalt_toolchains.gni

```diff
@@ -17,13 +17,15 @@ import("//build/toolchain/gcc_toolchain.gni")
 
 template("cobalt_clang_toolchain") {
   gcc_toolchain(target_name) {
-    forward_variables_from(invoker.variables,
+    forward_variables_from(invoker,
```

Comment:
Are any internal platforms that currently get built using this toolchain?

Reply:
no, ps4 uses its own toolchain, no other internal platform i can think of